osclib/stagingapi.py

```diff
@@ -1739,7 +1739,7 @@ def update_adi_frozenlinks(self, name, src_prj):
         l = ET.tostring(flink)
         http_PUT(url, data=l)
 
-    def create_adi_project(self, name, use_frozenlinks=False, src_prj=None):
+    def create_adi_project(self, name, use_frozenlinks=False, src_prj=None, nonfree_repo_required=False):
```

Comment:
nit: perhaps `nonfree` since the rest is implementation detail the outside does not need to know.